src/modules/launcher/Wox.Plugin/PluginMetadata.cs

```diff
@@ -58,6 +58,8 @@ internal set
 
         public bool IsGlobal { get; set; }
 
+        public bool ConfirmSys { get; set; }
```

Comment:
Let's remove it

Reply:
`ConfirmSys` is part of `AdditionalOptions` now

Reply:
Done